storage/queue.go

```diff
@@ -101,27 +128,23 @@ func (q *Queue) Add(ctx context.Context, e tessera.Entry) IndexFunc {
 //
 // To prevent blocking the queue longer than necessary, the notifications happen in a
 // separate goroutine.
```

Comment:
Is this comment still valid as the separate goroutine is now moved to the upper level?

Reply:
Great spot, thank you!
fixed.